### ISSUE TITLE
Incorrect WAVE results recorded

### ISSUE THREAD

**Issue by jared-w-smith** [alphagov/accessibility-tool-audit#3]
The following test case results are incorrectly coded as "Not found" for WAVE (my comments in parentheses):
- Large text does not have a contrast ratio of at least 3:1 so does not meet AA (you must select the Contrast tab in WAVE to see these identified)
- Large text does not have a contrast ratio of at least 4.5:1 so does not meet AAA (you must select the Contrast tab in WAVE to see these identified)
- html element is missing a lang attribute (identified at top of the page)
- Missing page title (identified at top of the page)
- Missing H1 (identified at top of the page)
- Table with column headers and double row headers (WAVE alerts to the presence of the headers)
- Links to a sound file, no transcript (WAVE identifies the media link with documentation to verify transcript)
- Empty label found

All of these cases are clearly identified by WAVE and should be corrected in the results and the summary descriptions/blog post.

**Comment by nickcolley** [comment]
Hey @jared-w-smith ! 

Thanks for the report, we'll try and address this as soon as possible.

In the meantime feel free to open a pull request if you want. 👍 

Thanks again 😸
